scripts/speaker_tasks/create_librispeech_alignment_manifest.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import random
+
+from nemo.collections.asr.parts.utils.manifest_utils import read_manifest, write_manifest
+
+random.seed(42)
+
+"""
+This script creates a manifest file containing word alignments.
+
+The alignments are obtained from: https://github.com/CorentinJ/librispeech-alignments
+"""
+
+# get librispeech examples without alignments for the desired dataset
+def get_unaligned_examples(unaligned_path, dataset):
```

Comment:
add at least one liner doc string on what function does, also explanation for input and return args